Relax tight dependencies that upstream uses to keep MSRV low

Upstream is concerned with keeping the MSRV low, but we don't need to
commit to that same approach to resolving the concern (debian archive
management should handle it for us).

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

--- a/Cargo.toml	2020-04-24 15:13:48.609357466 -0400
+++ b/Cargo.toml	2020-04-24 15:13:59.710831392 -0400
@@ -85,17 +85,17 @@
 version = "1"
 
 [dependencies.unicode-normalization]
-version = "= 0.1.9"
+version = "0.1.9"
 [dev-dependencies.quickcheck]
 version = "0.9"
 default-features = false
 
 [dev-dependencies.rand]
 version = "0.7"
 default-features = false
 
 [dev-dependencies.rpassword]
-version = "=4.0.3"
+version = "4.0.3"
 [build-dependencies.lalrpop]
 version = "0.17"
 
